src/Repository/IncentiveRepository.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\Incentive;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Persistence\ManagerRegistry;
+
+/**
+ * @method Incentive|null find($id, $lockMode = null, $lockVersion = null)
+ * @method Incentive|null findOneBy(array $criteria, array $orderBy = null)
+ * @method Incentive[]    findAll()
+ * @method Incentive[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class IncentiveRepository extends ServiceEntityRepository
```

Comment:
If there aren't any repository methods, can we remove this file (and the reference to it in the entity)?